docs/userGuide/usingComponents.md

```diff
@@ -205,4 +205,47 @@ If you want to include a keyword without rendering it to the page, add `d-none`
 This is good for catching <span class="keyword">regressions</span>
 ```
 
+# Using Tags
+
+You can use tags to selectively filter certain elements on web pages.
+
+Tags are specified by the `tags` attribute, **and can be attached to any HTML element**. For example:
+
+```html
+# Welcome to my site
+
+<p tags="language--english">Hello</p>
+<div tags="language--french">Bonjour</div>
+<span tags="language--spanish">Hola</span>
+```
+
+You need to specify the tags to include in `site.json`, under the `tags` option:
+
+```
+{
+  ...
+  "tags": ["language--english"]
+}
+```
+
+When compiled, only elements that match tags specified in the `site.json` files will be displayed. All other tagged elements will be hidden. In this case, only the element with the `language--english` tag will be displayed. This is helpful when creating multiple versions of a page without having to maintain separate copies. If this option is not specified, all tagged elements will be displayed.

```

Comment:
**During rendering**, only elements that match tags specified in the `site.json` files will be **rendered**. All other tagged elements will be **filtered out**. In this case, only the element with the `language--english` tag will be **rendered**.... If this option is not specified, all tagged elements will be **rendered**.